tests/test_pyrflx.py

```diff
@@ -550,17 +585,17 @@ def test_package_iterator(self) -> None:
 
     def test_tlv_get_first_unchecked_undefined(self) -> None:
         # pylint: disable=protected-access
-        self.assertEqual(self.tlv._get_first_unchecked("Length"), UNDEFINED)
+        self.assertEqual(self.tlv_checksum._get_first_unchecked("Length"), UNDEFINED)
 
     def test_check_nodes_opaque(self) -> None:
         # pylint: disable=protected-access
-        self.assertFalse(self.tlv._check_nodes_opaque("Length"))
-        self.assertTrue(self.tlv._check_nodes_opaque("Value"))
+        self.assertFalse(self.tlv_checksum._check_nodes_opaque("Length"))
+        self.assertTrue(self.tlv_checksum._check_nodes_opaque("Value"))
 
         self.frame.set("Destination", 2 ** 48 - 1)
         self.frame.set("Source", 0)
         self.frame.set("Type_Length_TPID", 1501)
-        self.frame._fields["Type_Length"].typeval.assign(1501, True)
+        self.frame._fields["Type_Length"].typeval.assign(1501)
```

Comment:
Why do you set a protected property here?

Reply:
This is used to test a path in `_is_valid_opaque_node` that is only accessible when the message is in an invalid state (with `Type_Length_TPID` set to 1501 `Type_Length` cannot be set with the `set` function).

Reply:
This sounds like a case which should usually never happen. Wouldn't it be better to add an assert in `_is_valid_opaque_node` for that? An assertion error could probably simplify finding a future implementation error.

Reply:
I don't think this test is needed. If i remove it, the test coverage does not go down. It probably is a leftover from a refactoring of the method.